pom.xml

```diff
@@ -44,6 +44,7 @@
         <che.docs.version>5.4.0-SNAPSHOT</che.docs.version>
         <che.lib.version>5.4.0-SNAPSHOT</che.lib.version>
         <che.version>5.4.0-SNAPSHOT</che.version>
+        <lsp4j.version>0.2.0-SNAPSHOT</lsp4j.version>
```

Comment:
@skabashnyuk I can't really split the PR: it would not even compile because the typefox interfaces are used everywhere, more or less.